src/solve.jl

```diff
@@ -1,17 +1,17 @@
-@inline (\)(a::StaticMatrix, b::StaticVecOrMat) = solve(Size(a), Size(b), a, b)
+@inline (\)(a::StaticMatrix, b::StaticVecOrMat) = _solve(Size(a), Size(b), a, b)
```

Comment:
By the way, I've renamed `solve` to `_solve` here just to make extra clear that it's an internal function.